gluoncv/model_zoo/simple_pose/pose_target.py

```diff
@@ -0,0 +1,85 @@
+"""
+Target generator for Simple Baselines for Human Pose Estimation and Tracking
+(https://arxiv.org/abs/1804.06208)
+"""
+import numpy as np
+
+
+class SimplePoseGaussianTargetGenerator(object):
+    """Gaussian heatmap target generator for simple pose.
+    Adapted from https://github.com/Microsoft/human-pose-estimation.pytorch
```

Comment:
if there is code copy-pasting we should keep the original header copyright

Reply:
will do